app/pages/home.cjsx

```diff
@@ -79,7 +79,7 @@ module.exports = React.createClass
     document.documentElement.classList.remove 'on-home-page'
 
   lastFourProjects: ->
-    @props.user.get("project_preferences", page_size: 4, sort: '+updated_at')
+    @props.user.get("project_preferences", page_size: 4, sort: 'updated_at')
```

Comment:
I just tested on my dev api node to see if the sort order was added using +, its not due to this [line](https://github.com/RestPack/restpack_serializer/blob/master/lib/restpack_serializer/options.rb#L65) leaving a space in the + param spot. Basically as the docs say, use a `-` to activate desc order otherwise if it's `sort=field` it'll be asc